.github/workflows/CI.yml

```diff
@@ -165,4 +165,35 @@ jobs:
       run: |
         # no need to 'snapcraft login' first, see https://forum.snapcraft.io/t/snapcraft-authentication-options/30473/21
         snapcraft upload *.snap --release=beta
+        
+  flatpak:
+    runs-on: ubuntu-22.04
 
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup .NET
+      run: |
+        # We need to install `ca-certificates`, otherwise we get these errors in the CI:
+        # Unable to load the service index for source https://api.nuget.org/v3/index.json.
+        # The SSL connection could not be established, see inner exception.
+        # The remote certificate is invalid because of errors in the certificate chain: UntrustedRoot
+        sudo apt install --yes --no-install-recommends ca-certificates
+        sudo apt install --yes --no-install-recommends dotnet7
+    - name: Install and setup Flatpak
+      run: |
+        sudo apt install --yes flatpak flatpak-builder
+        sudo flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
+        sudo flatpak install --noninteractive flathub org.freedesktop.Platform//23.08 org.freedesktop.Sdk//23.08
+        sudo flatpak install --noninteractive flathub org.freedesktop.Sdk.Extension.dotnet7//23.08
+    - name: Build project
+      run: |
+        cd src/PackWallet
+        dotnet publish -c Release PackWallet.fsproj
+        cd ../..
+    - name: Build flatpak package
+      run: flatpak-builder --disable-rofiles-fuse packwallet-build com.packwallet.yaml
+    - name: Test flatpak package
+      run: |
+        flatpak-builder --disable-rofiles-fuse --user --install --force-clean packwallet-build com.packwallet.yaml
+        flatpak run com.packwallet
```

Comment:
@webwarrior-ws pls move this test step to a new CI job that downloads artifact, to make sure it works without any other setup